### DISCUSSION TITLE
gif Support for setImage

### DISCUSSION THREAD

**Discussion by SpikeyMcHarten** [elgatosf/elgato#28] [Ideas]
Hi, i am very new to this discussion-platform, but i try to ask a question:
I do have a question about the gif-support in the 'setImage' Api-Function.

I read at
`https://www.reddit.com/r/StreamDeckSDK/comments/e0g0y6/when_i_use_setimage_with_an_animated_gif_i_only/`
that only the first image of a gif is rendered. I tried it out - it is still present and correct. I also read the workaround - but it doesn't satisfy me at all.

So can anybody tell me - how to push up this feature-request, or better: does anybody have a solution for sending animated gifs via api to streamdeck?

**Comment by ChekTek** [comment]
Hi SpikeyMcHarten,

I've created a feature request for our Stream Deck team!

Thanks
